crates/plume/src/lib.nr

```diff
@@ -109,7 +110,7 @@ fn sha256_12_coordinates(
         }
     }
 
-    sha256(res)
+    hash([to_field(res)], res.len())
 }
 
 #[test]
```

Comment:
Regarding all commented-out tests here: they need to be updated and passing `nargo test`, if uncommented. Currently, there are few data and compilation issues. Sorry for having to deal with commented code, I guess if noir'll support something akin to rust's `#[ignore]` macro in future, I'll update these

Reply:
No problem, feel free to update this branch too. I'm just an Aztec team member on my time off learning a bit about PLUME stuff 👍

Reply:
Sure, can you grant me write access to your fork?